src/components/PhotoItem.js

```diff
@@ -28,7 +28,7 @@ export default class PhotoItem extends Component {
     }).photo_name;
 
     return (
-      <div className='photo-item-wrapper'>
+      <div className='photo-item-wrapper' onClick={() => this.props.onClick()}>
```

Comment:
Why do we bind the parent onClick handler here?

Reply:
I want to make whole area of PhotoItem triggers display of PhotoModal.

Reply:
This is kind of breaking the single responsibility principle here.
Since 
1. the `onClick()` is just to call `displayPhotoModal` action
2. in `PhotoItem` you have enough data to pass into `displayPhotoModal`

Can we just make `PhotoItem` connect to redux and have its own `onClick()` handler, and in the handler, call `displayPhotoModal` ?

Reply:
I wanted to keep `PhotoItem` as a simple layout component, keep all handlers in `PhotoList`. It's totally doable through.

Reply:
But you've passed in `onClick()` as props already, that makes it no longer a simple layout component.

Reply:
You're absolutely right. Should I still bind, say, `handleClickPhotoItem` on `photo-item-wrapper`? Or should I create another div for clicking?

Reply:
Directly bind it on `photo-item-wrapper` should be fine. But either is fine.

Reply:
Fixed.